.gitignore

```diff
@@ -0,0 +1,48 @@
+# Logs
```

Comment:
A lot of these changes do not seem needed. I was able to achieve similar results with a smaller file:
`# Logs
logs
*.log
npm-debug.log*

# Runtime data
pids
*.pid
*.seed
*.pid.lock

# Compiled binary addons (http://nodejs.org/api/addons.html)
build/Release

# Dependency directories
node_modules
*/node_modules
jspm_packages
package-lock.json

# Optional npm cache directory
.npm

# Output of 'npm pack'
*.tgz
`